### DISCUSSION TITLE
what is the name of the MySQL DB in which all the messages are stored?

### DISCUSSION THREAD

**Discussion by wonderbee77** [postalserver/postal#2305] [Help with using Postal]
what is the name of the MySQL DB in which all the messages are stored?

**Comment by willpower232** [comment]
You should see that you have one database called `postal` which organises the app and the queue but you should also have many other databases which start with `postal-` and then a number which organise the data for each organisation within Postal.

You're likely to find the messages in the `postal-` ones but they can also briefly be in the main `postal` one. Also bear in mind that Postal by default does removes messages after a certain period of time so if you can't find the message in the UI then it won't be in the database.

**Comment by wonderbee77** [comment]
Thanks...
